src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,68 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
 import { Loader } from '../Loader';
+import { useAppSelector } from '../../app/hooks';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { actions as actionsActive } from '../../features/activeTodo';
 
 export const TodoModal: React.FC = () => {
+  const dispatch = useDispatch();
+  const activeTodo = useAppSelector(state => state.activeTodo);
+  const [users, setUsers] = useState<User>();
+
+  useEffect(() => {
+    if (activeTodo) {
+      getUser(activeTodo.userId)
```

Comment:
For each modal only one user always displayed:
![image](https://github.com/mate-academy/react_redux-list-of-todos/assets/86923089/e854eeae-994d-4e29-aa29-edb07426496f)